src/packages/core/encoding.cc

```diff
@@ -188,7 +189,7 @@ void f_buffer_transcode() {
   auto res = allocate_buffer(required);
 
   error_code = U_ZERO_ERROR;
-  auto written =
+  auto written [[gnu::unused]] =
```

Comment:
this is not unused, it is just because DEBUG_CHECK is ignored during RELEASE build.

Reply:
And exactly that is the reason to annotate it as 'possibly unused', that way we don't get a warning just because we do a release build...